### ISSUE TITLE
Pre-Averaged Filtered Data

### ISSUE THREAD

**Issue by aaprasad** [ColinConwell/DeepMouseTrap#3]
Hi, is there a way for me to get the data for this before averaging by trials but still with the same filtering calculations so it would be a tensor of shape (Num Neurons x Num Images x Num Trials x Num Responses)?

**Comment by ColinConwell** [comment]
This is possible, but it requires having downloaded the correct Brain Observatory data first. The unaveraged trial-level tensor is about 1.5GB larger than the averaged tensor. By same filtering calculations, I assume you mean reliability thresholding and the like?

**Comment by aaprasad** [comment]
Yes! the size isn't an issue for me but i wanted to calculate my own RDMs using crossnobis distance measures for which I believe I need the trial data as well

**Comment by aaprasad** [comment]
alternatively the trial averaged responses would actually be preferable if possible ie a tensor of shape Num Trials x Num Images x Num Neurons but either is fine
